apps/fxc-tiles/src/assets/airspaces/download.sh

```
@@ -1,10 +1,10 @@
 #! /usr/bin/env sh
 
-DIST_FOLDER=`realpath $(dirname "$0")`
-DST_FOLDER=`realpath $DIST_FOLDER/../../../apps/fxc-tiles/src/assets/airspaces`
+DIST_ASP_FOLDER=`realpath $(dirname "$0")`
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)